lib/routes/account.js

```diff
@@ -1477,6 +1478,8 @@ module.exports = (
               session.lastAccessTimeFormatted = localizeTimestamp.format(session.lastAccessTime,
                 request.headers['accept-language'])
 
+              session.os = session.uaOS || 'Unknown'
```

Comment:
> also saw `desktop` and `mobile` were not a localized.

Hmm, that's a good point. We do have access to localization in this repo now too, with the auth mailer stuff right? Maybe this repo should take responsibility for its own localizing everywhere?